server/src/test/java/org/elasticsearch/search/aggregations/bucket/missing/MissingAggregatorTests.java

```diff
@@ -91,7 +104,7 @@ public void testMatchNoDocs() throws IOException {
                     writer.addDocument(singleton(new SortedNumericDocValuesField(fieldType.name(), randomLong())));
                 }
             },
-            internalMissing -> {
+            (Consumer<InternalMissing>) internalMissing -> {
```

Comment:
I'm not sure about this, but could you write `(InternalMissing internalMissing) -> {` instead of the cast?

Reply:
That is much better, thanks